### PR TITLE
cephfs-shell: rewrite help text for put and get commands

### PR DESCRIPTION
**Pull request by rishabh-d-dave**
Just a quick fix.

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`

</details>

### CODE REVIEW

**Comment by rishabh-d-dave** [comment]
Do I need to create a tracker ticket for this?

**Comment by batrick** [comment]
> Do I need to create a tracker ticket for this?

no

File: src/tools/cephfs/cephfs-shell [hunk 1]
```diff
@@ -536,7 +536,8 @@ exists.')
     @with_argparser(put_parser)
     def do_put(self, args):
         """
-        Copy a file to Ceph File System from Local Directory.
+        Copy given file(s) from current working directory to CephFS. Files may
+        not necessarily be regular files; can be directories too.
```
Comment (batrick) [inline src/tools/cephfs/cephfs-shell]: I find `from current working directory` a bit redundant as that's always the case with any UNIX tool. How about `Copy a local file/directory to CephFS`

Comment (rishabh-d-dave) [reply src/tools/cephfs/cephfs-shell]: sounds great.

Comment (rishabh-d-dave) [reply src/tools/cephfs/cephfs-shell]: What about the other docstring -`Copy a file/directory from CephFS from to current working directory.`? Can't think of a way to not mention CWD here.

**Pull request merged**